app/assets/stylesheets/_manuals-print.scss

```diff
@@ -64,3 +64,17 @@ a {
     font-weight: bold;
   }
 }
+
+
+// TODO:
+// legislative lists are being published with hardcoded item numbers
+// when printing this results in doubled up numbering.
+// These styles hide browser numbering on these lists.
+// Ideally this would be a global solution not just in manuals-frontend
+.govuk-govspeak .legislative-list {
```

Comment:
That was something I thought about doing, I opted for limiting it to manuals rather than the govspeak component styles as we can't be sure on the way publishers are using them - so didn't want to globally strip the numbering, just limit it to cases we've seen where they are manually adding numbers / letters. Thoughts?